### DISCUSSION TITLE
Elasticsearch bulk update mode?

### DISCUSSION THREAD

**Discussion by jlazic** [vectordotdev/vector#17561] [General]
I was wondering why Vector does not support update action for bulk API? Documentation only mentions that index and create actions are supported, but could not find anything on why update action is not. Only thing that comes to mind is that document ID must be present for update action to work, but that's something we can ensure to have in VRL.

Having update bulk action would be very helpful for our use case where we append data to Elasticsearch document.

**Comment by jszwedko** [comment]
Thanks @jlazic ! Would you want to open a feature request for this? https://github.com/vectordotdev/vector/issues/new?assignees=&labels=type%3A+feature&projects=&template=feature.yml Understanding the use-case would be helpful.

**Reply by jlazic** [reply]
Sorry I missed your reply. Sent new feature request #17852
